x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc

```diff
@@ -1,136 +1,43 @@
 = X12 EDI Connector 2.5 - Examples - Mule 4
 
-The following examples show how to read, validate, and write EDI messages. In addition, you can send functional and interchange acknowlegments to your trading partners.
+Use this example to view the use case for Business-to-Business (B2B) EDI integration leveraging Anypoint X12 Connector. This example:
 
-== Read and Validate X12 EDI Messages
+* Transforms XML messages into Outbound EDI X12 850 purchase order transactions +
++
+* Transforms Inbound EDI X12 850 purchase orders into JSON +
++
+* Generates a Functional acknowledgment (997) +
 
-. To read an X12 message, search the palette for `X12 EDI:Configuration` and drag the X12 Read operation into a flow.
+An HTTPS Listener is used as the message source to receive the Enterprise purchase order XML messages from a back-end system. DataWeave and X12 Writes are used to translate the XML message into an X12 v4010 850 Purchase order EDI message. A (997) Functional Acknowledgment is generated upon receipt of the purchase order message.
```

Comment:
Suggestion to remove some of the passive voice:

An HTTPS Listener receives the...

DataWeave and X12 Writes translate the...

Reply:
Fixed, thanks.